webpack.config.js

```diff
@@ -154,13 +154,21 @@ if ( isDevelopment || isDesktop ) {
 const cssFilename = cssNameFromFilename( outputFilename );
 const cssChunkFilename = cssNameFromFilename( outputChunkFilename );
 
-const fileLoader = FileConfig.loader( {
-	// File-loader does not understand absolute paths so __dirname won't work.
-	// Build off `output.path` for a result like `/…/public/evergreen/../images/`.
-	outputPath: path.join( '..', 'images' ),
-	publicPath: '/calypso/images/',
-	emitFile: browserslistEnv === defaultBrowserslistEnv, // Only output files once.
-} );
+const fileLoader = FileConfig.loader(
+	// The server bundler express middleware server assets from the hard-coded publicPath `/calypso/evergreen/`.
+	// This is required so that running calypso via `npm start` doesn't break.
+	isDevelopment && ! isDesktop
+		? {
+				outputPath: 'images/',
+				publicPath: '/calypso/evergreen/images/',
```

Comment:
I don't like this much, but it seems necessary due to this: 
https://github.com/Automattic/wp-calypso/blob/53be70dae57843d43cedb841372c9fba6f4e8c6f/server/bundler/index.js#L104